### ISSUE TITLE
Fix "Links do not have a discernible name"

### ISSUE THREAD

**Issue by mhaack** [hlxsites/netcentric#192]
See accessibility issues on links, see https://pagespeed.web.dev/report?url=https%3A%2F%2Fwww.netcentric.biz%2F accesibility section.

**Comment by mhaack** [comment]
Some of them got already fixed by adding missing alt text for images with links.
